src/nbl/video/COpenGLFunctionTable.h

```diff
@@ -1626,8 +1642,9 @@ class COpenGLFunctionTable : public IOpenGL_FunctionTable
 			gl4Buffer.pglGetBufferSubData(GL_ARRAY_BUFFER, offset, size, data);
 			glBuffer.pglBindBuffer(GL_ARRAY_BUFFER, bound);
 		}
+		*/
 	}
-	void extGlVertexArrayAttribLFormat(GLuint vaobj, GLuint attribindex, GLint size, GLenum type, GLuint relativeoffset)
+	void extGlVertexArrayAttribLFormat(GLuint vaobj, GLuint attribindex, GLint size, GLenum type, GLuint relativeoffset) override
```

Comment:
please please please tell me you didnt use virtuals for the entire functions table!

Reply:
yeah i did, BUT:
virtualism is used only in classes like image, buffer (resources) etc and command buffer (not in queue and logical device)
It's easy to change though because i had in mind that you would like to change that. I would just have to make every COpenGL* class a template with function table type as a param

Reply:
function table absolutely shouldn't be full of virtuals, use CRTP or something

Reply:
assign to someone to fix it please @Crisspl